src/main/java/Duke.java

```diff
@@ -1,10 +1,83 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Task[] tasks = new Task[100];
+        String by = new String();
+        String at = new String();
+        int count = 0;
+        Scanner sc = new Scanner(System.in);
+        String greeting = "----------------------------------------------\n"
+                + "Hello! I'm Duke\n"
+                + "What can I do for you?\n"
+                + "----------------------------------------------\n";
+        String bye = "Bye. Hope to see you again soon!\n";
+        System.out.println(greeting);
+
+        while(true){
+            String userInput = sc.nextLine();
+            System.out.println("----------------------------------------------");
+            if(userInput.equals("bye")){
+                System.out.println(bye);
+                break;
+            }
+            else if(userInput.equals("list")){//show list
+                System.out.println("Here are the tasks in your list:");
+                for(int i = 0; i<count ; i++)
+                    System.out.println(i+1 + "." + tasks[i]);
+            }
+            else if(userInput.split(" ")[0].equals("mark")){//mark task
```

Comment:
under the coding standards, your else if should be written right next to the bracket above